### DISCUSSION TITLE
Upload freezes on .JP2 upload (iOS)

### DISCUSSION THREAD

**Discussion by brandonkal** [immich-app/immich#8607] [Feature Request]
### The bug

Upload process freezes at 100% on file ending with .JP2 extension

### The OS that Immich Server is running on

Synology DSM

### Version of Immich Server

v.1.101.0

### Version of Immich Mobile App

v1.101.0

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
NA
```

### Your .env content

```Shell
NA
```

### Reproduction steps

```bash
NA
```

### Additional information

Also fails on extension-less files (optimized files from DJI Fly App)

**Comment by mmomjian** [comment]
Immich [does not support](https://immich.app/docs/features/supported-formats/) JPEG2000.

**Comment by brandonkal** [comment]
Having one of those files on my device should not cause Immich to freeze and stop backing up photos

**Comment by mmomjian** [comment]
Can you please post some logs and screenshots of what happens as well as an example file? Please set the app logging level to "finest" before testing it.

**Comment by yagarea** [comment]
I am on version 1.123.0 on server and client and I have a same issue.

I tried to set log level to to `finest` but when I try to find `jp2` I have no matches. All I have is this in upload page:

![image](https://github.com/user-attachments/assets/2349e851-870c-4d13-9d69-8cf0e4207e22)

**Comment by cehwitham** [comment]
I am having a similar issue.

Immich: v1.124.0
App: 1.124.0

Foreground backup is running great until it gets to a file saved to the phone from the DJI Fly app. Any DJI optimized output JPEG seems to hang in the upload and I can't move on with the backup until I delete this file from my phone.

If the file format isn't supported, presumably the backup should skip the file?

I've looked in the logs but can't find anything relating to the files that are displayed when it hangs.
